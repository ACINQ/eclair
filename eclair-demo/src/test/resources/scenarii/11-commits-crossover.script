# Commits which cross over still get resolved.
# Initial state: A=1000000, B=0, both fee rates=10000
A:offer 500000000,b8928207364d445daa42f4ba8be0ef661b8d7190206c01f6ee8281566b3dec0a
B:recvoffer
A:commit
B:recvcommit
A:recvrevoke
B:commit
A:recvcommit
B:recvrevoke

B:fulfill 1,60303ae22b998861bce3b28f33eec1be758a213c86c93c076dbe9f558c11c752
B:commit
A:recvremove
A:recvcommit
B:recvrevoke
A:commit
B:recvcommit
A:recvrevoke

# Intermediate state: A=500000, B=500000, both fee rates=10000
echo ***A (intermediate)***
A:dump
echo ***B (intermediate)***
B:dump

A:offer 1000000,7b3d979ca8330a94fa7e9e1b466d8b99e0bcdea1ec90596c0dcc8d7ef6b4300c
B:offer 2000000,6016bcc377c93692f2fe19fbad47eee6fb8f4cc98c56e935db5edb69806d84f6
A:commit
B:commit

A:recvoffer
B:recvoffer
A:recvcommit
B:recvcommit

A:recvrevoke
B:recvrevoke

# They've got to come into sync eventually!
A:commit
B:commit
A:recvcommit
B:recvcommit
A:recvrevoke
B:recvrevoke

checksync
echo ***A***
A:dump
echo ***B***
B:dump
