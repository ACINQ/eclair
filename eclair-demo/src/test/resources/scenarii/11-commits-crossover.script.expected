***A (intermediate)***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 500000000
  Balance them: 500000000
  Fee rate: 10000
REMOTE COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 500000000
  Balance them: 500000000
  Fee rate: 10000
***B (intermediate)***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 500000000
  Balance them: 500000000
  Fee rate: 10000
REMOTE COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 500000000
  Balance them: 500000000
  Fee rate: 10000
***A***
LOCAL COMMITS:
 Commit 4:
  Offered htlcs: (2,1000000)
  Received htlcs: (1,2000000)
  Balance us: 499000000
  Balance them: 498000000
  Fee rate: 10000
REMOTE COMMITS:
 Commit 4:
  Offered htlcs: (1,2000000)
  Received htlcs: (2,1000000)
  Balance us: 498000000
  Balance them: 499000000
  Fee rate: 10000
***B***
LOCAL COMMITS:
 Commit 4:
  Offered htlcs: (1,2000000)
  Received htlcs: (2,1000000)
  Balance us: 498000000
  Balance them: 499000000
  Fee rate: 10000
REMOTE COMMITS:
 Commit 4:
  Offered htlcs: (2,1000000)
  Received htlcs: (1,2000000)
  Balance us: 499000000
  Balance them: 498000000
  Fee rate: 10000
